website/docs/r/organization.html.markdown

```diff
@@ -34,6 +34,7 @@ The following arguments are supported:
 * `collaborator_auth_policy` - (Optional) Authentication policy (`password`
   or `two_factor_mandatory`). Defaults to `password`.
 * `owners_team_saml_role_id` - (Optional) The name of the "owners" team.
+* `cost_estimation_enabled` - (Optional) Whether or not the cost estimation feature is enabled for all workspaces in the organization. Defaults to `false`.
```

Comment:
This was actually recently changed to `true`, and the documentation is out of date 😖  I've [opened a PR to get that changed](https://github.com/hashicorp/terraform-website/pull/1497), and provided that the documentation there doesn't need tweaking, we can put that here as well:

```suggestion
* `cost_estimation_enabled` - (Optional) Whether or not the cost estimation feature is enabled for all workspaces in the organization. Defaults to true on Terraform Cloud when the organization is on the Teams & Governance tier (else, the value is false and cannot be changed) and always true on Terraform Enterprise.
```

Reply:
> always true on Terraform Enterprise.

Ah I see, thanks for clarifying that point, since I was trying to observe the behavior (`1a` under the "Testing Plan").

Reply:
Gah, to be honest I had forgotten about the Terraform Enterprise Admin option you mention 😅 (I work primarily on Terraform Cloud).

There's yet one more caveat in that this value should behave as I mention above, _but_ in TFE even if it's `true`, the Admin setting you mention must also be enabled for runs to execute a cost estimate. I'm not sure if that detail is important here or not, but we could add it in:

```
In Terraform Cloud, defaults to true when the organization is on the Teams & Governance tier (else, the value is false and cannot be changed). Always defaults to true in Terraform Enterprise; Cost Estimation must also be enabled by Site Administrators.
```

Reply:
After a bit of worthsmithing, I think this is the clearest:

`Whether or not the cost estimation feature is enabled for all workspaces in the organization. Defaults to true. In Terraform Cloud, this value is always false and ignored if the organization does not have Teams & Governance features. In Terraform Enterprise, cost estimation must also be enabled in Site Administration.`

Reply:
Ok sure, but I'll defer changing this until you and @nfagerlund decide on the final copy?

Looks like he added a [new suggestion](https://github.com/hashicorp/terraform-website/pull/1497#discussion_r523266373) recently.